examples/llava/clip.cpp

```diff
@@ -752,13 +759,24 @@ static ggml_cgraph * clip_image_build_graph(clip_ctx * ctx, const clip_image_f32
         embeddings = ggml_add(ctx0, ggml_mul(ctx0, embeddings, model.pre_ln_w), model.pre_ln_b);
     }
 
+    std::vector<struct ggml_tensor *> embedding_stack;
+    // Check to see if we have 1+ set vision feature layers set; otherwise it's determined
+    // by the type of projector that this model has (usually last or second to last layer).
+    int max_feature_layer = get_deepest_feature_layer(ctx);
```

Comment:
small nits, but `max_feature_layer` is never changed for a given model, so I think it should be set on loading the model instead (i.e. `clip_model`)